### ISSUE TITLE
Adding uvloop support?

### ISSUE THREAD

**Issue by laky55555** [kristjanvalur/py-asynkit#1]
Hi, thank you for very nice async goodies :bow: 

Have you looked if it is possible to add [uvloop](https://github.com/MagicStack/uvloop/tree/master/uvloop) support?

**Comment by kristjanvalur** [comment]
No, not really.  What would you have in mind?  What kind of integration would be useful?

**Comment by laky55555** [comment]
As uvloop is significatly faster than built in loop, would be good if something like `SchedulingMixin` could be used with uvloop `Loop`. Not sure how feasible that is though.

**Comment by kristjanvalur** [comment]
I'll have to look at uvloop source.  Probably the only way is to make PRs for uvloop to add necessary utility functions.

**Comment by kristjanvalur** [comment]
I have a branch supporting uvloop now:  https://github.com/kristjanvalur/py-asynkit/tree/dev/uvloop
It uses a uvloop fork at https://github.com/kristjanvalur/uvloop/tree/kristjan/get_ready_queue

`asynkit.experimental.uvloop` can be imported and used instead of `uvloop`
